modules/tide_share_link/src/Form/ShareLinkTokenForm.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Drupal\tide_share_link\Form;
+
+use Drupal\Core\Entity\ContentEntityForm;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Render\Element;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Form controller for Share Link Token edit forms.
+ *
+ * @ingroup tide_share_link
+ */
+class ShareLinkTokenForm extends ContentEntityForm {
+
+  /**
+   * The current user account.
+   *
+   * @var \Drupal\Core\Session\AccountProxyInterface
+   */
+  protected $account;
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container) {
+    // Instantiates this form class.
+    $instance = parent::create($container);
+    $instance->account = $container->get('current_user');
+    return $instance;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    /** @var \Drupal\tide_share_link\Entity\ShareLinkTokenInterface $entity */
+    $form = parent::buildForm($form, $form_state);
+    // Set the field description to the timestamp value widget.
+    // @see https://www.drupal.org/project/drupal/issues/2508866
+    if (!empty($form['expiry']['widget']['#description']) && isset($form['expiry']['widget'])) {
+      $first_key = array_key_first(Element::children($form['expiry']['widget']));
```

Comment:
`array_key_first()` is only available from PHP 7.3, so we need https://github.com/dpc-sdp/dev-tools/pull/43 so that phpcs doesn't fail code linting.